geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -372,7 +376,7 @@ public void testScenario4() throws Exception {
 
       current_fullpath = jtaObj.currRegion.getFullPath();
       assertEquals("failed retirieving current region fullpath after txn rollback",
-          "/" + DEFAULT_RGN + "/region1", current_fullpath);
+          SEPARATOR + DEFAULT_RGN + "/region1", current_fullpath);
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?